### ISSUE TITLE
Remove/merge nightly branch when we move prod builder to new CI

### ISSUE THREAD

**Issue by axk4545** [RITlug/TigerOS#80]
Reminder issue to track progress of CI and remember to remove/merge nightly branch once the CI is stable and ready to deploy to our main build server

**Comment by ct-martin** [comment]
I'm closing this issue as we have now adopted the usage of feature branches and should consider the deletion of these branches when we merge them as part of the workflow (as opposed to making issues for each branch). I also added a reminder about this on the development subsection homepage on the wiki. Should a branch not get deleted when it gets merged, then opening an issue would be appropriate (to also verify with those working in it that it is no longer needed), but otherwise I'd rather we don't fill the issue tracker with issues blocked by major parts of the development.
